src/relax/transform/decompose_composite_ops.cc

```diff
@@ -110,21 +111,60 @@ class NormInferenceSimplifier : public ExprMutator {
   Map<Expr, Expr> batch_norm_map_;
 };
 
+class OpDecomposer : public ExprMutator {
```

Comment:
The name of OpDecoposer is too generic since it's only for `tensor_to_shape`

Reply:
We can extend this function to add other composite ops like `erf`, `attention` which will be upcoming soon.